src/Options/index.js

```diff
@@ -103,6 +103,12 @@ export interface ParseServerOptions {
   databaseOptions: ?DatabaseOptions;
   /* Adapter module for the database; any options that are not explicitly described here are passed directly to the database client. */
   databaseAdapter: ?Adapter<StorageAdapter>;
+  /* Disable case insensitivity (collation) on queries and indexes, needed if you use MongoDB serverless or AWS DocumentDB. */
+  disableCollation: ?boolean;
+  /* Transform Email to lowercase on create/update/login/signup. On queries client needs to ensure to send Email in lowercase format. */
+  transformEmailToLowerCase: ?boolean;
+  /* Transform Username to lowercase on create/update/login/signup. On queries client needs to ensure to send Username in lowercase format. */
+  transformUsernameToLowerCase: ?boolean;
```

Comment:
```suggestion
  /* Optional. If set to `true`, the `email` property of a user is automatically converted to lowercase before being stored in the database. Consequently, queries must match the case as stored in the database, which would be lowercase in this scenario. If `false`, the `email` property is stored as set, without any case modifications. Default is `false`.
  :DEFAULT: false */
  convertEmailToLowercase: ?boolean;
   /* Optional. If set to `true`, the `username` property of a user is automatically converted to lowercase before being stored in the database. Consequently, queries must match the case as stored in the database, which would be lowercase in this scenario. If `false`, the `username` property is stored as set, without any case modifications. Default is `false`.
  :DEFAULT: false */
  convertUsernameToLowercase: ?boolean;
```

- Extended the description.
- Added missing `:DEFAULT:` values; these guarantee that also in code there is always a default value and the properties are never `undefined` - consider in code and tests if necessary.
- A simply change in case is more commonly referred to as a "conversion" in technical literature.
- Removed uppercase in var name `...ToLowerCase`, as "lowercase" is 1 word.